cmd/minikube/cmd/addons/list.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package addons
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"k8s.io/minikube/pkg/minikube/cluster"
+)
+
+var addonsListCmd = &cobra.Command{
+	Use:   "list",
+	Short: "Lists all available minikube addons as well as there current status (enabled/disabled)",
+	Long: `Lists the ADDON_NAME addons value to ADDON_VALUE
+	These values can be overwritten by flags or environment variables at runtime.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) != 0 {
+			fmt.Fprintln(os.Stderr, "usage: minikube addons list")
+			os.Exit(1)
+		}
+		err := list()
+		if err != nil {
+			fmt.Fprintln(os.Stderr, err)
+			os.Exit(1)
+		}
+	},
+}
+
+func init() {
+	AddonsCmd.AddCommand(addonsListCmd)
+}
+
+func stringFromStatus(addonStatus bool) string {
+	if addonStatus {
+		return "enabled"
+	}
+	return "disabled"
+}
+
+func list() error {
+	for addonName, addonBundle := range cluster.Addons {
+		addonStatus, err := addonBundle.IsEnabled()
+		if err != nil {
+			return err
+		}
+		fmt.Printf("- %s: %s\n", addonName, stringFromStatus(addonStatus))
```

Comment:
Could we do the go struct thing with a template format string here, like in status?